okhttp-logging-interceptor/src/main/java/okhttp3/logging/HttpLoggingInterceptor.java

```diff
@@ -257,7 +273,12 @@ public Level getLevel() {
           logger.log(buffer.clone().readString(charset));
```

Comment:
small nit – we’re cloning the buffer here, but that isn’t necessary if it was gzipped. (The difference is negligible.)